content/en/infrastructure/containers/configuration.md

```diff
@@ -192,6 +192,3 @@ Set the environment variable on both the Process Agent and Cluster Agent contain
 ## Further reading
 
 {{< partial name="whats-next/whats-next.html" >}}
-
-[1]: https://app.datadoghq.com/containers
-[2]: /infrastructure/containers
```

Comment:
It's my linter in VS Code that did this 😕 I think it's because there's already links using 1 / 2 ? https://github.com/DataDog/documentation/blob/ac580f01a149300f32feda9394ee56ac50120e98/content/en/infrastructure/containers/configuration.md?plain=1#L176-L177

maybe it should be 3 / 4?